main.go

```diff
@@ -445,6 +439,51 @@ func (o *Options) Run() error { //nolint:gocyclo
 	return nil
 }
 
+func (o *Options) loadProwJobs(dbc *db.DB, sippyConfig v1.SippyConfig) []error {
+
+	allErrs := []error{}
+
+	gcsClient, err := gcs.NewGCSClient(context.TODO(),
+		o.GoogleServiceAccountCredentialFile,
+		o.GoogleOAuthClientCredentialFile,
+	)
+	if err != nil {
+		log.WithError(err).Error("CRITICAL error getting GCS client which prevents importing prow jobs")
+		allErrs = append(allErrs, err)
+		return allErrs
+	}
+
+	var bigQueryClient *bigquery.Client
+	if o.LoadOpenShiftCIBigQuery {
+		bigQueryClient, err = bigquery.NewClient(context.Background(), "openshift-gce-devel",
+			option.WithCredentialsFile(o.GoogleServiceAccountCredentialFile))
+		// TODO: support GoogleOAuthClientCredentailFile like above for GCS?
```

Comment:
Anyone know if this is needed?

Reply:
It's a second auth option but I don't think we've ever used it